core/templates/pages/android-page/android-page.component.html

```diff
@@ -0,0 +1,185 @@
+      <div class="oppia-android-key-feature-container">
+        <div class="oppia-android-key-feature-description">
+          <div *ngIf="featuresShown > 0" [@fadeIn] class="oppia-android-key-feature">
+            <div class="oppia-android-key-feature-text oppia-android-text-color">Learn through engaging stories</div>
+            <div class="oppia-android-key-feature-subtext">And try our helpful hints to guide you along</div>
+          </div>
+          <div *ngIf="featuresShown > 1" [@fadeIn] class="oppia-android-key-feature">
+            <div class="oppia-android-key-feature-text oppia-android-text-color">Learn anytime, anywhere</div>
+            <div class="oppia-android-key-feature-subtext">Play lessons even without an Internet connection.</div>
+          </div>
+          <div *ngIf="featuresShown > 2" [@fadeIn] class="oppia-android-key-feature">
+            <div class="oppia-android-key-feature-text oppia-android-text-color">Learn in your language</div>
+            <div class="oppia-android-key-feature-subtext">
+              For now, the app is available in <strong>English</strong>, <strong>Brazilian</strong><br>
+              <strong>Portuguese</strong>, and
+              <strong> Swahili</strong>.<br>
+              <br>
+              More languages will be added soon!
+            </div>
+          </div>
+          <div *ngIf="featuresShown > 3" [@fadeIn] class="oppia-android-key-feature">
+            <div class="oppia-android-key-feature-text oppia-android-text-color">Switch between learners</div>
+            <div class="oppia-android-key-feature-subtext">Create and maintain up to 10 profiles on one device.</div>
+          </div>
+        </div>
+        <div class="oppia-android-key-feature-image-container oppia-web-only">
+          <img *ngIf="featuresShown === 1" [@fadeIn] class="oppia-android-key-feature-image" src="/assets/images/android/android-landing-1.svg" alt="">
+          <img *ngIf="featuresShown === 2 || featuresShown === 3" [@fadeIn] class="oppia-android-key-feature-image" src="/assets/images/android/android-landing-2.svg" alt="">
+          <img *ngIf="featuresShown >= 4" [@fadeIn] class="oppia-android-key-feature-image" src="/assets/images/android/android-landing-4.svg" alt="">
+        </div>
+      </div>
+    </div>
+    <div *ngIf="!androidUpdatesSectionIsSeen" class="oppia-web-only">
+      <div #feature1 class="oppia-android-empty-div"></div>
+      <div class="oppia-android-empty-div"></div>
+      <div #feature2 class="oppia-android-empty-div"></div>
+      <div class="oppia-android-empty-div"></div>
+      <div #feature3 class="oppia-android-empty-div"></div>
+      <div class="oppia-android-empty-div"></div>
+      <div #feature4 class="oppia-android-empty-div"></div>
+      <div class="oppia-android-empty-div"></div>
+    </div>
+    <div #androidUpdatesSection class="oppia-android-app-updates-container oppia-android-app-updates-background">
+      <div *ngIf="false" class="d-flex">
```

Comment:
This section is hidden intentionally because we do not have the backend parts yet for updating the mailing list with new users. This will be done in a follow-up PR.

Reply:
/cc @aks681 as FYI